omod/src/main/webapp/resources/scripts/fragments/patientdashboard/encountertemplate/defaultEncounterTemplate.js

```diff
@@ -52,9 +52,9 @@ $(function() {
 	    		if(encounterDetailsSection.html() == "") { encounterDetailsSection.html("<i class=\"icon-spinner icon-spin icon-2x pull-left\"></i>");}
 	    jq.getJSON(
 	        		emr.fragmentActionLink("htmlformentryui", "htmlform/viewEncounterWithHtmlForm", "getAsHtml", { encounterId: id })
-	        ).success(function(data){
+	        ).done(function(data){
 	            encounterDetailsSection.html(data.html);
-	        }).error(function(err){
```

Comment:
Is this change related to what this ticket is all about?

Reply:
The deprecation of success() to done() came together with the deprecation of error () to fail().....so it seemed logical to do both in the same sitting....What would be your suggestion?....Should I modified the ticket description to accommodate both or I shud have a different ticket for that too or I ignore it all together?

Reply:
If you want both, then you would need to update both the ticket description and commit message to reflect what you are doing

Reply:
Let me modify then... Thanks